sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Tests/DocumentTests/ActivityEventTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Internals.DataCollection;
+using Azure.Monitor.OpenTelemetry.LiveMetrics.Models;
+using OpenTelemetry.Trace;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Tests.DocumentTests
+{
+    public class ActivityEventTests : DocumentTestBase
+    {
+        public ActivityEventTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void VerifyActivityWithExceptions()
+        {
+            // SETUP
+            var uniqueTestId = Guid.NewGuid();
+
+            var activitySourceName = $"activitySourceName{uniqueTestId}";
+            using var activitySource = new ActivitySource(activitySourceName);
+            // TODO: Replace this ActivityListener with an OpenTelemetry provider.
+            var listener = new ActivityListener
```

Comment:
@TimothyMothra - Why not just use the SDK here?

Reply:
Other tests in the live metrics follow the same pattern. As a follow up in later PRs this could be modified to use SDK to keep it simple.

Reply:
Do you mean OTel/InMemoryExporter?

That introduces additional overhead. This test needs only an Activity to test the `DocumentHelper` conversion methods.

Reply:
he meant Sdk.CreateTracerProvider() instead of creating a listener.

Reply:
This listener is listening to everything, that could be problematic. We should dispose it as well.